ush/rocoto/setup_expt_fcstonly.py

```diff
@@ -115,6 +127,48 @@ def edit_baseconfig():
     comrot = args.comrot if args.comrot is None else os.path.join(args.comrot, pslot)
     expdir = args.expdir if args.expdir is None else os.path.join(args.expdir, pslot)
     gfs_cyc = args.gfs_cyc
+    partition = args.partition
+
+    # Set machine defaults
+    if machine is 'WCOSS_DELL_P3':
+      base_git = '/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git'
+      base_svn = '/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git'
+      dmpdir = '/gpfs/dell3/emc/global/dump'
+      nwprod = '/gpfs/dell1/nco/ops/nwprod'
+      homedir = '/gpfs/dell2/emc/modeling/noscrub/$USER'
+      stmp = '/gpfs/dell3/stmp/$USER'
+      ptmp = '/gpfs/dell3/ptmp/$USER'
+      noscrub = '/gpfs/dell2/emc/modeling/noscrub/$USER'
+      account = 'GFS-DEV'
+      queue = 'dev'
+      queue_arch = 'dev_transfer'
+      if partition is not None and partition in ['3p5']:
```

Comment:
`if partition is not None` is not required here, because if it is `None`, the `partition in ['3p5']` will evaluate to `False`.

Reply:
Thanks, I'll remove that bit.